tools/boostsecurity

```diff
@@ -0,0 +1,69 @@
+{
+      "name": "Boostsecurity.io",
+      "publisher": "Boostsecurity.io",
+      "description": "The zero friction platform for AppSec Testing and Application Security Posture Management (ASPM) securing both Software Supply Chains and AI Development.",
+      "website_url": "https://boostsecurity.io/",
+      "capabilities": [
+        "SBOM"
+      ],
+      "availability": [
+        "COMMERCIAL_LICENSE",
+        "SUBSCRIPTION"
+      ],
+      "functions": [
+        "ANALYSIS",
+        "AUTHOR",
+        "DISTRIBUTE",
+        "PACKAGE_MANAGER_INTEGRATION",
+        "TRANSFORM"
+      ],
+      "analysis": [
+        "LICENSE_REPORTING",
+        "OUTDATED_COMPONENTS",
+        "POLICY_EVALUATION",
+        "SECURITY_VULNERABILITIES"
+      ],
+      "transform": [
+        "BOM_STANDARD"
+      ],
+      "packaging": [
+        "APPLICATION",
+        "COMMAND_LINE_UTILITY",
```

Comment:
i dont see howany of these packaging methods come. you are providing a web-based service, right?